packages/govtnz-ds-website/src/utils/index.ts

```diff
@@ -0,0 +1,45 @@
+export function generatePrintLinksList() {
+  const existingWrapper = document.querySelector(
+    '.external-links-print-wrapper'
+  );
+
+  // Exit early if the user has already printed and generated the links
+  if (existingWrapper) return;
+
+  const mainContent = document.querySelector('main');
+  if (!mainContent) return;
+
+  const links = Array.from(mainContent.querySelectorAll('a'));
```

Comment:
The `Array.from(...` is the bit I needed to add for IE11. The NodeList (and `forEach` thereof) wasn't working. Tested and working in IE11 now.